recipes/jupytext/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jupytext" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 05203c7efc0c7de6947a401538045a42deddbe0b01ed304905f1c9e293d0e742
+
+build:
+  number: 0
+  entry_points:
+    - jupytext = jupytext.cli:jupytext
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - mock
```

Comment:
Is this really needed for install?

Reply:
@mwouts, you can probably answer that better than me

Reply:
mock is required only when jupytext runs. And actually, only when the tests are executed, and when Jupytext's plugin for Jupyter is used.

Reply:
Ok , so you don't need it in host or run, but in `tests: requires:`